FDRS_Sensor/fdrs_sensor.h

```diff
@@ -176,16 +187,143 @@ void beginFDRS() {
 
 }
 
-void transmitLoRa(uint8_t* mac, DataReading * packet, uint8_t len) {
+//  USED to get ACKs from LoRa gateway at this point.  May be used in the future to get other data
+// Return type is crcResult struct - CRC_OK, CRC_BAD, CRC_NULL.  CRC_NULL used for non-ack data
+crcResult getLoRa() {
+#ifdef USE_LORA
+  int packetSize = LoRa.parsePacket();
+  if ((packetSize - 6) % sizeof(SystemPacket) == 0 && packetSize > 0) {  // packet size should be 6 bytes plus multiple of size of SystemPacket
```

Comment:
i hadn't thought of that! This means we can send multiple SystemPackets at once, right?

Reply:
Yep!  Just like DataReading.

Reply:
Out of the blue, I realized that I think misunderstood something here.

If we are receiving data and it could be either a DataReading or a SystemPacket, we can't rely on dividing by the size of the structure to distinguish between the two, since there could be common divisors. DataReading being 7 bytes makes that a bit less likely, but it could also change. If I'm not confused (which I could be) we will still have to send SystemPackets one at a time and check for them by seeing if they're smaller than DataReadings.

This code is fine, though.

Reply:
Yeah, that's right, there are two types of data being sent between devices - DataReadings and SystemPackets.  At this point I believe the only way to distinguish the two different types is by the size of the packet.   If the structure of one or both of those types changes we'll have to change the code like you mentioned.   I'm not sure I know of another way to distinguish between the two other than size at this point.  An alternative would be to add another byte to each packet to distinguish between the two types.   Possibly do something with the CRC calculation but that would be starting to get confusing.   I'm open to ideas if you think of alternatives.

Reply:
Distinguishing by size is okay, That's how I'd planned it. The question about images this morning got me thinking about a third type that could be a full 250 bytes. Now, this could currently be a distinguishing feature since the most DataReadings could take up is 245 bytes. But if DataReading was 10 bytes, then we could have an issue when there are 25 items.

So... Maybe we lower our max packet size for DataReadings to 249 and introduce a new BulkPacket that uses exactly 250. 
The bulk packet will have a reading id (or something),  and individual frame numbers for each packet. The frames, when pieced together, will form an actual image file, sound file, etc.

There's a lot to do first, but I wanted to share that idea with you for the future. I've been talking about SystemPackets for like, six months and just implemented them a couple weeks ago,😄